src/plugins/line-numbers/prism-line-numbers.ts

```diff
@@ -272,3 +273,14 @@ export default {
 		return combineCallbacks(removeListener, completeHook);
 	},
 } as PluginProto<'line-numbers'>;
+
+export default Self;
+
+if (typeof document !== 'undefined') {
```

Comment:
Without this check, Rollup chokes on bundling the code (when splitting it into chunks): in `documentReady`, it tries to access a property of an undefined `document`.